esp-box/include/bsp/esp-box.h

```diff
@@ -273,6 +274,9 @@ esp_err_t bsp_spiffs_unmount(void);
 #define BSP_LCD_PIXEL_CLOCK_HZ     (40 * 1000 * 1000)
 #define BSP_LCD_SPI_NUM            (SPI3_HOST)
 
+#define BSP_LCD_DRAW_BUF_HEIGHT     (10)
```

Comment:
This changes the buffer height from 50 to 10. Was this supposed to be part of the commit related to home button handling?

Reply:
@igrr Thank you for this point. I overlooked it - I don't understand, why it was changed in this PR. It is working with 50 too.

Reply:
Because this bsp is mainly adapted to esp-idf 5.0. Compared with 4.4, the sram reserved for application is smaller. 
50 lines will lead to insufficient sram, and the program cannot run normally now.

Reply:
Sounds fair, but could you put that change into a separate commit, with the message explaining why this is changed?  It would help someone looking at `git blame` output later if this change isn't part of an unrelated commit.

Reply:
ok, i'll rearrange it, thanks.